site/_scss/globals/_common-blocks.scss

```diff
@@ -0,0 +1,19 @@
+// Common Blocks
```

Comment:
Let's create a new directory block-groups and put this inside there, so `block-groups/_common.scss` and expand the comment to explain why this selection of blocks is considered common.